lib/new-utils.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const Long = require("long");
+
+// maxInt value is based on how does Long splits values between internal high and low fields.
+const maxInt = BigInt(0x100000000);
+const minusOne = BigInt(-1);
+
+/**
+ * Converts from bigint provided by napi into Long type used by the datastax library
+ * BigInt is the way napi handles values too big for js Number type,
+ * while Long is the way datastax parts of the code handle 64-bit integers.
+ * @param {bigint} from
+ * @returns {Long}
+ */
+function bigint_to_long(from) {
```

Comment:
Shouldn't we keep the convention of the Datastax driver to write functions in camelCase?

Reply:
We could as well use a different case to clearly distinguish Datastax procedures and entities from ours. This is an open discussion.